app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -19,6 +21,9 @@ class BrowserApplication : Application() {
         // We want the log messages of all builds to go to Android logcat
         Log.addSink(AndroidLogSink())
 
+        Glean.initialize(this)
```

Comment:
Do we always need to initialize Glean? If `BuildConfig.TELEMETRY_ENABLED` is false even?

Reply:
Yeah, I think so. We will still call into `Glean` here and there (e.g. adding an event) even though at the end of the day we will not upload anything.

@Dexterp37 Correct? You always want us to initialize Glean, right?

Reply:
It will also make it easier to handle the preference in settings: We just need to enable/disable and not care about whether everything was initialized already or not.

Reply:
> Yeah, I think so. We will still call into `Glean` here and there (e.g. adding an event) even though at the end of the day we will not upload anything.
> 
> @Dexterp37 Correct? You always want us to initialize Glean, right?

Mh, this is a good question. I don't think we *require* you to do so. In fact, if you don't need telemetry, skipping on glean init will also skip on some baseline ping data collection that will eventually not be used.